blueprints/component-test/mocha-files/__root__/__testType__/__path__/__test__.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'chai';
```

Comment:
This entire file should be replaced with the mocha rfc232 file!

Reply:
This is still wrong—until you replace it with the correct file, stuff is going to be broken. Also, while you're at it, go ahead and remove all the shenanigans around supporting unit tests. As noted on the QUnit test blueprint, we *only* care here about integration tests, esp. b/c unit tests straight-up don't work with Glimmer components (as it should be!).